src/components/Card/Card.stories.mdx

```diff
@@ -53,25 +58,39 @@ import { getCategory } from "../../utils/componentCategories";
 | Component Version | DS Version |
 | ----------------- | ---------- |
 | Added             | `0.24.0`   |
+| Latest            | `0.24.0`   |
 
 <Description of={Card} />
 
-Cards are contained units of information that can stand independently from other elements for context. While a Card is a self-contained unit, Cards are often grouped together and displayed in some type of grid layout.
+To quote Material Design: "_A card is identifiable as a single, contained unit. A card can stand alone, without relying on surrounding elements for content._" Additionally, while a `Card` is a self-contained unit, `Card` components are often grouped together and displayed in some type of grid layout.
+
+The DS `Card` component is a highly configurable, composable component. It can be shown with or without an image and the content area is composable with Card-specific child components: `CardHeading`, `CardActions` and `CardContent`. These child components can be arranged in any order and each child component may be used multiple times within the content area.
+
+#### CardHeading
+
+The `CardHeading` component mirrors the standard DS `Heading` component and accepts the [same props](https://nypl.github.io/nypl-design-system/storybook-static/?path=/docs/heading--heading-props).
+
+#### CardActions
+
+The `CardActions` component is used to display CTAs for the card. `Button` and `Link` components should be passed as children into the `CardActions` component.
```

Comment:
Write out CTA before it's abbreviated.

Reply:
Maybe we also need a page with terms and design language used in the DS.

Reply:
Updated.  And I will add a ticket for that.